src/posts/2021-09-27-Using-focus-style-for-accessibility

```diff
@@ -0,0 +1,133 @@
+---
+title: Using focus style for accessibility
+description: "A simple guide on how to make your website more accessible with focus style."
+category: How-to
+author: Ezekiel Lawson
+date: 2021-09-27 
+further_reading:
+  - title: Style focus
+    url: https://https://web.dev/style-focus/
+    source:  Web Dev
+    year: 2018
+  - title: Designing usable focus indicators
+    url: https://www.deque.com/blog/give-site-focus-tips-designing-usable-focus-indicators/
+    source:  Caitlin Geier
+    year: 2016
+  - title: Never remove CSS outlines
+    url: https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/
+    source: Guilherme Simões
+    year: 2013
+tags:
+  - How-to
+---
+
+While accessing any digital interface, a focus indicator is important it highlights the active intractable component in a webpage. 
+
+## What is a focus style?
+
+A focus is a pseudo-class used to indicate or highlights an intractable component in a webpage.
```

Comment:
>A focus

A focus what?

Reply:
Can we link "a pseudo-class" to [this MDN article on `:focus`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus). That will help explain the jargon to someone not familiar with the concept.